src/cleanvision/issue_managers/image_property_issue_manager.py

```diff
@@ -96,13 +134,40 @@ def find_issues(
         to_be_computed = list(set(self.issue_types).difference(defer_set))
         raw_scores: Dict[str, Any] = {issue_type: [] for issue_type in to_be_computed}
         if to_be_computed:
-            for path in tqdm(filepaths):
-                image = Image.open(path)
-                for issue_type in to_be_computed:
-                    raw_scores[issue_type].append(
-                        self.image_properties[issue_type].calculate(image)
+            if n_jobs is None:
+                if psutil_exists:
+                    n_jobs = psutil.cpu_count(logical=False)  # physical cores
+                if not n_jobs:
+                    # either psutil does not exist
+                    # or psutil can return None when physical cores cannot be determined
+                    # switch to logical cores
+                    n_jobs = multiprocessing.cpu_count()
+
+            if n_jobs == 1:
```

Comment:
For this issue manager too, follow the same pattern of code as it would reduce duplicate code.
1. create a base method that would take in path, to_compute and any other required arguments
2. write a wrapper function around it to be used in `imap_unordered()`
3. Collect results